Workade/Views&ViewModels/Magazine/MagazineViewController.swift

```diff
@@ -0,0 +1,216 @@
+        
+        return button
+    }()
+    
+    private let line: UIView = {
+        let line = UIView()
+        line.backgroundColor = .gray
+        line.translatesAutoresizingMaskIntoConstraints = false
+        
+        return line
+    }()
+    
+    private var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.backgroundColor = .white
+        stackView.distribution = .fillEqually
+        
+        return stackView
+    }()
+    
+    var detailView = UIViewController()
+    
+    private var selectedTab = 0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        detailView = TotalDetailViewController()
+        
+        settingStackView()
+        setupLayout()
+        settingDetailView()
+    }
+    
+    // MARK: AutoLayout 설정
+    private func setupLayout() {
+        view.addSubview(viewTitle)
+        let viewTitleLayout = [
+            viewTitle.topAnchor.constraint(equalTo: view.topAnchor, constant: 20),
+            viewTitle.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20)
+        ]
+        
+        view.addSubview(stackView)
+        let stackViewLayout = [
+            stackView.topAnchor.constraint(equalTo: viewTitle.bottomAnchor, constant: 14),
+            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20)
+            ]
+        
+        view.addSubview(line)
+        let lineLayout = [
+            line.topAnchor.constraint(equalTo: stackView.bottomAnchor, constant: 10),
+            line.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            line.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            line.heightAnchor.constraint(equalToConstant: 2)
+        ]
+        
+        NSLayoutConstraint.activate(viewTitleLayout)
+        NSLayoutConstraint.activate(stackViewLayout)
+        NSLayoutConstraint.activate(lineLayout)
+    }
+    
+    func settingDetailView() {
+        for view in self.view.subviews {
+            view.removeFromSuperview()
+        }
+        
+        setupLayout()
+        
+        self.addChild(detailView)
+        self.view.addSubview(detailView.view)
+        detailView.didMove(toParent: self)
+        
+        detailView.view.translatesAutoresizingMaskIntoConstraints = false
+        let detailViewLayout = [
+            detailView.view.topAnchor.constraint(equalTo: line.bottomAnchor, constant: 10),
+            detailView.view.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            detailView.view.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ]
+        NSLayoutConstraint.activate(detailViewLayout)
+    }
+    
+    func settingStackView() {
```

Comment:
```suggestion
    func setupStackView() {
```
[P3] 문맥 통일성을 위해서 위와 같은 수정은 어떤가요?

Reply:
반영하였습니다 👍